src/LiveComponent/doc/index.rst

```diff
@@ -183,36 +183,23 @@ let's keep going because… things get cooler.
 
 Hooks: Handle Component Behavior
 --------------------------------
-Most of the time, you'll just pass data to your components and 
-let it handle the rest. However, if you need to do something 
-more complex during certain stages of a component's lifecycle, you 
-can take advantage of lifecycle hooks.
+Most of the time, you'll just pass data to your components and let it handle the rest. However, if you need to do something more complex during certain stages of a component's lifecycle, you can take advantage of lifecycle hooks.
```

Comment:
```suggestion
Most of the time, you'll just pass data to your components and let it handle the
rest. However, if you need to do something more complex during certain stages of
a component's lifecycle, you can take advantage of lifecycle hooks.
```

Symfony documentation standards has line length requirements: https://symfony.com/doc/current/contributing/documentation/standards.html

Could you update the other changes to stay under 80 chars lines ? 

Thank you!